nssp_demo/collate_score_tables.R

```diff
@@ -0,0 +1,202 @@
+script_packages <- c(
+  "data.table",
+  "argparser",
+  "stringr"
+)
+
+## load in packages without messages
+purrr::walk(script_packages, \(pkg) {
+  suppressPackageStartupMessages(
+    library(pkg, character.only = TRUE)
+  )
+})
+
+
+#' Get all the subdirectories within a parent directory
+#' that match the pattern for a forecast run for a
+#' given disease and optionally a given report date.
+#'
+#' @param parent_dir Directory in which to look for forecast subdirectories.
+#' @param diseases Names of the diseases to match, as a vector of strings,
+#' or a single disease as a string.
+#' @return A vector of paths to the forecast subdirectories.
+get_all_forecast_dirs <- function(dir_of_forecast_date_dirs,
```

Comment:
candidate for `hewr`

Reply:
Agree. Out of scope for now I think, though.

Reply:
Created #113